<ui version="4.0" >
 <class>TopLevelScreen</class>
 <widget class="QWidget" name="TopLevelScreen" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NULL</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="ScreenTitle" native="1" name="screentitle" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="printerUrl" >
       <property name="text" >
        <string>http://NPI0040CA990285.test.com</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="printer_IPv4" >
       <property name="text" >
        <string>http://16.23.86.65</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="maximumSize" >
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textToMenu" >
         <property name="text" >
          <string>Press # to enter menu...</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="manufacturerLogo" >
         <property name="maximumSize" >
          <size>
           <width>62</width>
           <height>75</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap>../../../../../local/CProjects/Samson/resources/hp_logo.bmp</pixmap>
         </property>
         <property name="scaledContents" >
          <bool>true</bool>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>-3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenTitle</class>
   <extends>QWidget</extends>
   <header>ScreenTitle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../guidocs/images/turbotaj/turbotaj_image_resource.qrc" />
 </resources>
 <connections/>
</ui>
